java/test/src/main/java/org/ray/api/test/MultiThreadingTest.java

```diff
@@ -42,6 +43,32 @@ public Integer echo(int num) {
     }
   }
 
+  @RayRemote
+  public static class ActorIdTester {
+
+    public ActorIdTester() {
+      Assert.assertNotEquals(Ray.getRuntimeContext().getCurrentActorId(), ActorId.NIL);
```

Comment:
we can save the result of `Ray.getRuntimeContext().getCurrentActorId()` in a var, and check it later in `getCurrentActorId`.

Reply:
Done.